extensions/usememos/src/lib/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface PreferenceValues {
+  memosServerUrl: string;
+  memosServerToken: string;
+  model: string;
+  language: string;
+  openAiApiKey?: string;
+  openAiBasePath?: string;
+}
+
```

Comment:
```suggestion
```
Already added in raycast-env.d.ts

Reply:
Alright, I just realized that I can use this here. I will make the correction immediately.